File: connectors/Arbbet.Python.Connectors/Arbbet.Domain/Entities/Sport.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using Arbbet.Domain.Bases;
using Arbbet.Domain.Enums;
using Arbbet.Domain.Interfaces;

namespace Arbbet.Domain.Entities
{
  public class Sport : AUnifiedEntity<Sport>, IIdentifiable, ICodeNamed, IUnifiedEntity<Sport>
  {
    public string Name { get; set; }

    public string Code { get; set; }

    public virtual IList<Competition> Competitions { get; set; }
  }
}

File: connectors/Arbbet.Python.Connectors/Arbbet.Connectors/Business/Sport.py
class Sport(object):
    def __init__(self, id, code, name, idfdj):
        self.id = id
        self.code = code
        self.name = name
        self.idfdj = idfdj

    @property
    def id(self):
        return self._id

    @id.setter
    def id(self, value):
        self._id = value

    @property
    def name(self):
        return self._name

    @name.setter
    def name(self, value):
        self._name = value

    @property
    def code(self):
        return self._code

    @code.setter
    def code(self, value):
        self._code = value

    @property
    def idfdj(self):
        return self._idfdj

    @idfdj.setter
    def idfdj(self, value):
        self._idfdj = value